cg/meta/upload/mutant/mutant.py

```diff
@@ -0,0 +1,18 @@
+from click import Context
+
+from cg.meta.upload.upload_api import UploadAPI
+from cg.meta.workflow.mutant import MutantAnalysisAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Analysis, Case
+
+
+class MutantUploadAPI(UploadAPI):
+
+    def __init__(self, config: CGConfig):
+        self.analysis_api: MutantAnalysisAPI = MutantAnalysisAPI(config)
+        super().__init__(config=config, analysis_api=self.analysis_api)
+
+    def upload(self, ctx: Context, case: Case, restart: bool) -> None:
+        self.upload_files_to_customer_inbox(case)
+        latest_analysis: Analysis = case.analyses[0]
+        self.update_uploaded_at(latest_analysis)
```

Comment:
Not specified in the refinement but should the upload_started_at be set as well?

Reply:
You can check the other pipelines and see where the uploaded_started_at is set. If the mutant upload falls into the same category of workflows as the others I think it should mimic that.

The raw data workflow has a special flow and logic, wheres MIP-dna and Balsamic are similar.